pip/models/__init__.py

```diff
@@ -1,4 +0,0 @@
-from pip.models.index import Index, PyPI
```

Comment:
You might have removed 2 files unintentionally. 

`pip.models.__init__` and `pip._vendor.cachecontrol.compat`.

Reply:
Both of those files still exists.